pyproject.toml

```diff
@@ -206,8 +206,9 @@ packaging = ">=16.1"
 # At the time of writing, we only use functions from the version `importlib.metadata`
 # which shipped in Python 3.8. This corresponds to version 1.4 of the backport.
 importlib_metadata = { version = ">=1.4", python = "<3.8" }
-# This is the most recent version of Pydantic with available on common distros.
-pydantic = ">=1.7.4"
+# This is the most recent version of Pydantic with available on common distros
+# We are currently incompatible with >=2.0.0: https://github.com/matrix-org/synapse/issues/15858
+pydantic = "<2.0.0"
```

Comment:
I am pretty sure we still want the version to be `>=1.7.4`.
You should probably use `^=1.7.4` instead as suggested in the bug, which means `please take the higher version that should be API-compatible to my minimum requirement`, which in this case translate to `>=1.7.4 AND <2.0.0`.

Reply:
And thanks for your contribution by the way :) .

Reply:
Will make the change shortly!

Reply:
Done!